app/_data/docs_nav_ee_0.31-x.yml

```diff
@@ -64,13 +64,15 @@
 
 - title: Guides &amp; References
   items:
-    - text: Kong Implementation Checklist
-      url: /kong-implementation-checklist
-    - text: Changelog
+    - text: Kong Enterprise Changelog
       url: /docs/enterprise/changelog
       absolute_url: true
     - text: Architecture Overview
       url: /kong-architecture-overview
+    - text: Architecture Patterns
```

Comment:
would recommend nesting into Architecture and then having two sections: overview and patterns. But would advise keeping the url for overview since it's been a page so long and should keep SEO.